client/analysisui/layoutcell.js

```diff
@@ -146,8 +144,9 @@ const LayoutCell = function(parent) {
         if (this.$content) {
             if (this.$content.css === undefined)
                 this.$content = null;
-            this.$content.css( "position", "absolute");
-            this.$el.append(this.$content);
+            else
+                this.$content.css( "position", "absolute");
+                this.$el.append(this.$content);
```

Comment:
single quotes, and this either requires braces, or the indentation is wrong. i suspect the former.

Reply:
good catch

Reply:
k pushed the fix